### ISSUE TITLE
Sprint 22: Egg Sheeran 🍳 

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#9899]
## ⚛️  React

- [ ] #9901 Tooltip @joshblack @dakahn 
  - [x] #9941
  - [x] #9966 Ship new version of Tooltip with updated Popover styles
  - [ ] #9967 Ship `IconButton` after `Tooltip` is merged
  - [x] #9902
- [ ] #8487  @abbeyhrt @tay1orjones 
  - [ ]  https://github.com/carbon-design-system/carbon/issues/9988
  - [x] https://github.com/carbon-design-system/carbon/issues/9991
- [x] #9712
  - [x] #10009 @jnm2377 
  - [x] #10010 @jnm2377 
  - [x] #10012
  - [x] #10013
  - [x] #10014
  - [x] #10015
  - [x] #10017
  - [x] #10019
  - [x] #10021
  - [ ] #10022 @jnm2377 
- [x] #9535 
  - [ ] Review what's left and break down into actionable parts @joshblack @tay1orjones 
- [ ] #9948 @tay1orjones 

## 🚢  Migration

- [ ] #9640 @jnm2377 @joshblack @tay1orjones 
  - [ ] Implement workflow for detecting the v11 upgrade path in a workspace and updating dependencies accordingly

## 🎨  Styles
- [x] https://github.com/carbon-design-system/carbon/issues/9888 @sstrubberg 
- [x] #9947 @tay1orjones 

## 📝  Website
**Enhancements:**
- [ ] v11: Disclosure pattern docs (design crit edits and open PR) @laurenmrice 
- [ ] v11: Toggletip component docs (Make PR) @kingtraceyj 
- [ ] v11: Animate usage images @johnbister 
  - [ ] Disclosure gifs 
  - [ ] Tooltip gif
- [ ] Clean up and update Status indicators icons (assets and library) @thyhmdo 
  - [ ] Alert colors (with Diana S.)
- [ ] Progress bar component docs @thyhmdo 

## 🎨 Design Kit

**Sketch**
- [ ] v11: Create layer input symbols in all 4 themes. Find and replace existing layer instances of fields + border bottom and replace with layer-field symbol @aagonzales 
- [ ] v11: Double check components are using the correct tokens after token migration  @aagonzales
- [x] v11: Add Popover component @laurenmrice 

**Figma**
- [ ] Figma – component reviews (on-going) @aagonzales @laurenmrice 
- [ ] Release Figma Carbon White theme kit @aagonzales @laurenmrice 

## 💬 Support

- [x] #9686 @abbeyhrt
- [x] #9922 @abbeyhrt 
- [ ] #9879 @tay1orjones 
- [x] https://github.com/carbon-design-system/carbon/issues/9895 @dakahn 
- [ ] https://github.com/carbon-design-system/carbon/issues/4812 @dakahn 
- [x] https://github.com/carbon-design-system/carbon/issues/9750 @jnm2377 
- [x] https://github.com/carbon-design-system/carbon/issues/9804 @sstrubberg 

## 🚀  Releases

- [x] #10024 @tay1orjones 
- [x] https://github.com/carbon-design-system/carbon/issues/10084 @dakahn @sstrubberg 

## 🐝  Brand

- [ ] https://github.com/carbon-design-system/design-language-website/issues/999 @aledavila 
- [x] https://github.com/carbon-design-system/design-language-website/issues/1009 @aledavila 
- [x] https://github.com/carbon-design-system/design-language-website/issues/1010 @jnm2377  
- [x] Update IDL with latest theme version @aledavila 

## 🖥️ DPO
- [x] Racial Equity Fathom tracking bug https://github.ibm.com/Design/racial-equity-in-design/issues/285 @sstrubberg 
- [x] W3 design tile update https://github.ibm.com/Design/w3-design/issues/1079 @alisonjoseph 
- [ ] EDT toolkit spanish file upload (waiting on files) @alisonjoseph 

## 📈     Dataviz
- [x] Visual Chart images (with Eliad) @thyhmdo 
- [x] Accessibility enhancements/fixes
- [ ] Heatmap (PR ready, should be merged in a week)
- [ ] Zoombar enhancements (PR ready, should be merged in a week)
- [ ] (STRETCH GOAL) New Carbon theming updates (moving to next sprint)

## 📣 Communication

- [ ] Toggle tip/Tooltip/Disclosures/Popover Medium article (finish images) @kingtraceyj 

## 🧙‍♂️ Meeting facilitators
- Backlog Grooming @aledavila 
- Sprint Retro @jnm2377 
- Sprint Planning (next sprint 22)
  - Dev/System: @tay1orjones

**Comment by theiliad** [comment]
## 📈 Dataviz
Some of these might carry over

- [ ] Accessibility enhancements/fixes
- [ ] New Carbon theming updates
- [ ] Heatmap
- [ ] Zoombar enhancements
